ipn/ipnlocal/local.go

```diff
@@ -3434,6 +3471,21 @@ func (b *LocalBackend) setNetMapLocked(nm *netmap.NetworkMap) {
 			delete(b.nodeByAddr, k)
 		}
 	}
+
+	handlePorts := make([]uint16, 0, 4)
+	if b.ShouldRunSSH() {
+		handlePorts = append(handlePorts, 22)
+	}
+	for _, domain := range nm.DNS.CertDomains {
+		if v, err := b.store.ReadState(ipn.StateKey(fmt.Sprintf("_handle-tcp/%s", domain))); err == nil {
```

Comment:
I don't know what you are doing here with the state store. This needs comments.

Reply:
For now only the CLI debug command. The CLI will come later.

The format is described only in https://docs.google.com/document/d/1c09VN98W77mf-qpZKD8tukqn7QNvT68kRaoqLsd8byE/edit# for now.

Reply:
rest of the code is fine, but I think this should go in the same PR as the thing that writes this state.

Reply:
done

Reply:
Thanks, I can reason about this now!